libyul/optimiser/ConditionalSimplifier.cpp

```diff
@@ -72,12 +72,12 @@ void ConditionalSimplifier::operator()(Block& _block)
 				)
 				{
 					YulString condition = std::get<Identifier>(*_if.condition).name;
-					langutil::SourceLocation location = _if.location;
+					std::shared_ptr<DebugData const> debugData = _if.debugData;
```

Comment:
I'd say this local variable is pretty much useless.

Reply:
I'll remove it